builder-image/README.md

```diff
@@ -3,8 +3,10 @@ Build with docker:
 
 * JDK11: `cekit --redhat build docker`
 * JDK17: `cekit --redhat build --overrides image-jdk17-overrides.yaml docker`
+* JDK21: `cekit --redhat build --overrides image-jdk21-overrides.yaml podman`
 
 Build with OSBS:
 
 * JDK11: `cekit --redhat build --overrides rh-jdk11-overrides.yaml osbs`
```

Comment:
not related to the PR but this line that go away

Reply:
Removed. Thanks.